bears/__init__.py

```diff
@@ -20,3 +20,25 @@ def assert_supported_version():  # pragma: no cover
     if sys.version_info < (3, 4):
         print('coala supports only python 3.4 or later.')
         exit(4)
+
+
+def check_coala_version():
+    """
+    Check the installed coala and coala-bears version when the coalib loads
+    coala-bears module.
+    """
+    try:
+        import coalib
+        if coalib.__version__ != __version__:
```

Comment:
@jayvdb 

> coala-bears version might be greater than coala version.

Could you explain further about the version comparison case? AFAIU, coala use `MAJOR.MINOR.PATCH.BUILD`. Do we need to just compare that `MAJOR.MINOR` of coalib is **greater** than bears?

```python
coalib_version = '.'.join(coalib.__version__.split('.')[:-2])
bears_version = '.'.join(bears.__version__.split('.')[:-2])
if StrictVersion(coalib_version) > StrictVersion(bears_version):
    print('warning')
```